# What is VCF Annotation
VCF (Variant Call Format) annotation is a process that involves adding additional information to genetic variants listed in a VCF file. This information can include functional consequences, population frequency data, known disease associations, and more.

# Keywords
- **VCF Annotation tools**
- **VCF Annotation database**
- **Genetic variants annotation**
- **Variant annotation**
- **Genetic testing**


# Tools

## 1. ANNOVAR 
A versatile tool for functionally annotating genetic variants.
- **[GitHub](https://github.com/clindet/anor)**  (**⭐ 32 Stars**) • [**ClinDet**](https://github.com/clindet/) 
- **[Docker](https://registry.hub.docker.com/r/bioinstaller/annovarr)** (**⤓ 324 Pulls**)
- ---
- **[GitHub](https://github.com/dceoy/annovar-vcf-cli)**  (**⭐ 0 Stars**) • [**dceoy**](https://github.com/dceoy) 
- **[Docker](https://registry.hub.docker.com/r/bioinstaller/annovarr)** (**⤓ 324 Pulls**)
- **[Docker](https://hub.docker.com/r/bioinfochrustrasbourg/annovar)** (**⤓ 100k+ Pulls**)
  
## 2. SnpEff
Provides detailed functional annotations for genetic variants.

- **[GitHub](https://github.com/pcingola/SnpEff)**  (**⭐ 232 Stars**) • [**pcingola**](https://github.com/pcingola/) 
- **[Docker](https://hub.docker.com/r/nfcore/snpeff)** (**⤓ 10k+ Pulls**)

## 3. VEP (Variant Effect Predictor)
Developed by Ensembl, it predicts the functional effects of variants.

- **[GitHub](https://github.com/Ensembl/ensembl-vep)**  (**⭐ 432 Stars**) • [**Ensembl**](https://github.com/Ensembl) 
- **[Docker](https://hub.docker.com/r/ensemblorg/ensembl-vep)** (**⤓ 10M+ Pulls**)

## 4. BCFtools
 A set of utilities for manipulating VCF and BCF files.
 
 - **[GitHub](https://github.com/samtools/bcftools)**  (**⭐ 624 Stars**) • [**samtools**](https://github.com/samtools/) 
- **[Docker](https://hub.docker.com/r/biocontainers/bcftools)** (**⤓ 1M+ Pulls**)

## 5. ClinVar Mine
A tool specifically for annotating variants with clinical significance using ClinVar data.

- **[GitHub](https://github.com/ncbi/clinvar)**  (**⭐ 58 Stars**) • [**ncbi**](https://github.com/ncbi) 


## 6. GEMINI
Integrates VCF files with a wide range of annotation sources.

- **[GitHub](https://github.com/arq5x/gemini)**  (**⭐ 316 Stars**) • [**arq5x**](https://github.com/arq5x) 
- **[Docker](https://hub.docker.com/r/sibswiss/gemini)** (**⤓ 80 Pulls**)

## 7. Vcfanno
Annotates VCF files with information from BED, BAM, and VCF files.

- **[GitHub](https://github.com/brentp/vcfanno)**  (**⭐ 350 Stars**) • [**brentp**](https://github.com/brentp) 
- **[Docker](https://hub.docker.com/r/clinicalgenomics/vcfanno)** (**⤓ 100k+ Pulls**)


## 8. VCFtools
Provides tools for a wide range of VCF file manipulations.

- **[GitHub](https://github.com/vcftools/vcftools)**  (**⭐ 481 Stars**) • [**vcftools**](https://github.com/vcftools) 
- **[Docker](https://hub.docker.com/r/biocontainers/vcftools)** (**⤓ 100k+ Pulls**)

# Databases
Each tool has specific databases that it can use for annotation. Commonly used databases include:

- **Population Databases:** 1000 Genomes, ExAC, gnomAD
- **Functional Databases:** dbSNP, ClinVar, COSMIC
- **Gene Annotations:** refGene, ensGene, knownGene
- **Regulatory Databases:** ENCODE, Roadmap Epigenomics



